### ISSUE TITLE
Proto: file "messages.proto" is already registered

### ISSUE THREAD

**Issue by chung-chao** [hashgraph/hedera-sdk-go#275]
### Description

We were going to implement Hedera Token Services. After installation of Hedera Golang SDK, we met with fatal error about protobuffer that keeping us from starting our server.

We expected to run very well, so we use the workaround specified by [Google](https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict) so far. In this report, we would like to know any solution except for the workaround 

![image](https://user-images.githubusercontent.com/78942782/132156653-2bb7f1bf-6513-4a20-a382-248e2e5fe5eb.png)

### Steps to reproduce

1. Ran `main.go` to start server. Everything looked well
2. Installed `hedera-sdk-go` 
3. Ran `go mod tidy`
4. Started server again. Met with fatal error.

### Additional context

1. Started our server
![image](https://user-images.githubusercontent.com/78942782/132155859-f174a85e-a2f3-4631-85ab-1897eeb25bf2.png)
```bash
> go run main.go
```

2. Installed Hedera Golang SDK
![image](https://user-images.githubusercontent.com/78942782/132156096-993683f1-41de-4588-922e-30ee187f447b.png)

```bash
> go get github.com/hashgraph/hedera-sdk-go/v2
```
3. Ran `go mod tidy`
![image](https://user-images.githubusercontent.com/78942782/132156119-86785215-2fb2-45a5-b03a-7f49a27b110c.png)

```bash
> go mod tidy
```

4. Met with `file "messages.proto" is already registered`
![image](https://user-images.githubusercontent.com/78942782/132156154-9bfd48b9-d36a-4876-828a-554fc2ccb791.png)

5. Searched on this [link](https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict). However, we thought that it might not be the best workaround to us.

6. Screenshot of `go.mod`
![image](https://user-images.githubusercontent.com/78942782/132160312-ece0852c-b0f1-40d1-9924-12d6a099b8c7.png)

### Hedera network

testnet

### Version

go version go1.16 darwin/amd64, github.com/hashgraph/hedera-sdk-go/v2 v2.1.16

### Operating system

macOS

**Comment by danielakhterov** [comment]
Do you have another dependency that is using protobufs as well? I'm not even sure if that could cause this issue, but I created a sample test application which depends on the current SDK and it seems to work fine.

**Comment by chung-chao** [comment]
Base on the message above, I think it is [gozilliqa-sdk@1.2.0](https://pkg.go.dev/github.com/Zilliqa/gozilliqa-sdk@v1.2.0/protobuf).
What's weird on our end is that it worked very well before installing Hedera SDK.

I also noticed that when I install the SDK, it also upgraded [uuid](github.com/google/uuid) and then asked me to do furthermore `go mod` as screenshot 2nd. Then, it crashed as before.
![image](https://user-images.githubusercontent.com/78942782/132981949-8df32d25-1303-4f67-8140-1db3da923274.png)
![image](https://user-images.githubusercontent.com/78942782/132982019-3e651a85-9c57-42dc-800d-f23cb165bd27.png)

Sorry I'm not very familiar with Golang. Hope that can be figured well

**Comment by danielakhterov** [comment]
Are you able to show what `main.go` contains? I've added both `gozilliqa-sdk` and `uuid` with no issues in my example. Are you declaring your own protobufs in your project?

**Comment by chung-chao** [comment]
@danielakhterov Yes. For the `main.go`, it contains as following:
```golang
package main

import (
	"app/config"
	"app/initializer"
)

func main() {
	configs := config.LoadEnv().Database
	server := initializer.Server{}
	server.InitializeDB(
		configs.Driver,
		configs.Username,
		configs.Password,
		configs.Port,
		configs.Host,
		configs.DbName,
	)
	server.InitializeRoutes()
	server.Run()
}
```

For the `app/initializer`, it contains as following. Hope it helps!
```golang
package initializer

import (
	"app/authentication"
	"app/logger"
	"app/models"
	"fmt"

	"github.com/go-martini/martini"
	"github.com/martini-contrib/render"

	"gorm.io/driver/mysql"
	"gorm.io/gorm"
)

type Server struct {
	DB            *gorm.DB
	Martini       *martini.ClassicMartini
}

// .....
func (s *Server) InitializeDB(Dbdriver, DbUser, DbPassword, DbPort, DbHost, DbName string) {
	// ......
}

func (s *Server) InitializeRoutes() {
	// ....
}

func (s *Server) Run() {
	s.Martini.Run()
}
```

> Are you declaring your own protobufs in your project?
For sure that we didn't declare our own protobufs. It's simply for some internal usage, so we didn't even touch that.

For the reproducing, yeah we'll spend extra time to work on that.

Hope the above example code helps!

**Comment by danielakhterov** [comment]
I saw that stack overflow post before, but I just couldn't understand how using protobuf v1.26.1 would have a fixed but, but protobuf v1.27.0 would not. Nonetheless, it is good that fix works. As for if there are vulnerabilities with using a slightly older version of protobuf I don't think there are. It just might have less features and/or change how the generated code looks, but that should not cause any vulnerability issues from my understanding.
